build-locally.py

```diff
@@ -51,6 +55,14 @@ def verify_config(ns):
 def main(args=None):
     p = ArgumentParser("build-locally")
     p.add_argument("config", default=None, nargs="?")
+    p.add_argument(
+        "--debug",
+        action="store_true",
+        help="Setup debug environment using `conda debug`",
+    )
+    p.add_argument(
+        "--output-id", help="If running debug, specifiy the output to setup."
```

Comment:
```suggestion
        "--output-id", help="If running debug, specify the output to setup."
```
I'm assuming this comes from a template somewhere that needs to be fixed?

Reply:
Yep. Good catch though, we completely ignore these part of the PR as we assume they are "the template." See https://github.com/conda-forge/conda-smithy/pull/1403 for the fix.